engines/catalog/spec/spec_helper.rb

```diff
@@ -1,5 +1,3 @@
 # frozen_string_literal: true
 
 require '../../spec/spec_helper'
-
-Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].sort.each { |f| require f }
```

Comment:
👀  Hmm, I didn't know about this engine!
I see that it doesn't require any spec support files 👍